packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -354,23 +354,27 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
    */
   private async setAnsName(): Promise<void> {
     if (this._network?.chainId && this._account) {
-      if (this._account.ansName) return;
-      // ANS supports only MAINNET or TESTNET
-      if (
-        !ChainIdToAnsSupportedNetworkMap[this._network.chainId] ||
-        !isAptosNetwork(this._network)
-      ) {
-        this._account.ansName = undefined;
-        return;
-      }
+      try {
+        if (this._account.ansName) return;
+        // ANS supports only MAINNET or TESTNET
+        if (
+          !ChainIdToAnsSupportedNetworkMap[this._network.chainId] ||
+          !isAptosNetwork(this._network)
+        ) {
+          this._account.ansName = undefined;
+          return;
+        }
 
-      const aptosConfig = getAptosConfig(this._network, this._dappConfig);
-      const aptos = new Aptos(aptosConfig);
-      const name = await aptos.ans.getPrimaryName({
-        address: this._account.address.toString(),
-      });
+        const aptosConfig = getAptosConfig(this._network, this._dappConfig);
+        const aptos = new Aptos(aptosConfig);
+        const name = await aptos.ans.getPrimaryName({
+          address: this._account.address.toString(),
+        });
 
-      this._account.ansName = name;
+        this._account.ansName = name;
+      } catch (error: any) {
+        console.log(`Error setting ANS name ${error}`);
+      }
```

Comment:
If i understand correctly, we wrapped with try/catch here?
The error is just console logged, I don't think this adds any value so let's remove it.

Or.. what were you trying to do? Maybe there's some other way

Reply:
It happened when I had a bad connection, and because we dont catch the error it throws and breaks the app. So added a try/catch but I dont want to throw because it is not really an error, I just want the app to not break

Reply:
ok would you mind wrapping only the section around `getPrimaryName` then?

Reply:
```typescript
try {
  this._account.ansName = await aptos.ans.getPrimaryName({
    address: this._account.address.toString(),
  });
} catch (err) {
  // either ignore or console log
}
```